src/components/modal/modal.js

```diff
@@ -79,17 +79,16 @@ export default class Modal {
     }
   }
 
-  makePageContentInert() {
+  toggleModalAndPageAriaHidden() {
```

Comment:
We don't need to do any of this as the `dialog` handles all of the background page state. What you originally changed was correct, to remove it all,

Reply:
Ah ok ill remove the commits